qiita_db/test/test_study.py

```diff
@@ -179,6 +179,54 @@ def _change_processed_data_status(self, new_status):
             "UPDATE qiita.processed_data SET processed_data_status_id = %s",
             (id_status,))
 
+    def test_get_info(self):
```

Comment:
Can you add another test in which there is more than 1 study? You can use the study create to add a new study (note that if the creation fails, we already have tests for the create function that will fail....)